definitions/output/all/reprocess_pages.js

```diff
@@ -0,0 +1,262 @@
+    '$.PageSpeed',
+    '$.rank',
+    '$.sessionstorage_size',
+    '$.startedDateTime',
+    '$.url',
+    '$.urlhash',
+    '$.urlShort',
+    '$.usertiming',
+    '$.wptid',
+    '$.wptrun'
+  ) AS summary,
+  STRUCT<
+    a11y JSON,
+    cms JSON,
+    cookies JSON,
+    css_variables JSON,
+    ecommerce JSON,
+    element_count JSON,
+    javascript JSON,
+    markup JSON,
+    media JSON,
+    origin_trials JSON,
+    performance JSON,
+    privacy JSON,
+    responsive_images JSON,
+    robots_txt JSON,
+    security JSON,
+    structured_data JSON,
+    third_parties JSON,
+    well_known JSON,
+    wpt_bodies JSON,
+    other JSON
+  >(
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.a11y"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.cms"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.cookies"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.css-variables"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.ecommerce"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.element_count"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.javascript"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.markup"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.media"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.origin-trials"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.performance"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.privacy"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.responsive_images"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.robots_txt"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.security"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.structured-data"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.third-parties"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.well-known"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.wpt_bodies"),
+    JSON_REMOVE(
+      SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), 
+      '$.a11y', 
+      '$.cms', 
+      '$.cookies', 
+      '$.css-variables', 
+      '$.ecommerce', 
+      '$.element_count', 
+      '$.javascript', 
+      '$.markup', 
+      '$.media', 
+      '$.origin-trials', 
+      '$.performance', 
+      '$.privacy', 
+      '$.responsive_images', 
+      '$.robots_txt', 
+      '$.security', 
+      '$.structured-data', 
+      '$.third-parties', 
+      '$.well-known', 
+      '$.wpt_bodies'
```

Comment:
@tunetheweb Should we replace `summary` cleanup with `custom_metrics` cleanup?
```suggestion
      '$.wpt_bodies',
      '$.Colordepth',
      '$.avg_dom_depth',
      '$.doctype',
      '$.document_height',
      '$.document_width',
      '$.inline_style_bytes',
      '$.localstorage_size',
      '$.meta_viewport',
      '$.num_iframes',
      '$.num_scripts',
      '$.num_scripts_async',
      '$.num_scripts_sync',
      '$.sessionstorage_size',
      '$.usertiming',
```

Reply:
As I said above, while they are kind like custom metrics-style info, I don't want to add them to custom_metrics column and create confusion when people go to the https://github.com/HTTPArchive/custom-metrics to find the definition and can't.

Reply:
1. They are custom metrics defined here: https://github.com/HTTPArchive/custom-metrics/tree/main/dist
2. They are currently available in 3 places:
```sql
SELECT 
  JSON_QUERY(custom_metrics, "$.avg_dom_depth"),
  JSON_QUERY(payload, "$._avg_dom_depth"),
  JSON_QUERY(summary, "$.avg_dom_depth")
FROM `httparchive.sample_data.pages_10k`
WHERE date = "2024-09-01"
  AND client = "mobile"
  AND rank <= 1000
```
3. I suggest to keep only `summary.avg_dom_depth`, and remove other instances.

Reply:
Ohhh in that case they are custom metrics! Apologies!!!

So I suggest we only keep `custom_metrics.avg_dom_depth`